app/forms/providers/means/identify_income_form.rb

```diff
@@ -0,0 +1,100 @@
+module Providers
+  module Means
+    class IdentifyIncomeForm
```

Comment:
**NOT a blocker** Just from a naming point of view, and following on from the controller renaming comment,  it might be clearer to name it IdentifyRegularIncomesForm. Have followed a similar naming in the CFE branch

Reply:
In the spirit of the other comment, I'd probably lean toward something like `RegularIncomeForm`.

Does that convey the intent more accurately from your perspective?